src/migrations/20241007072653_create_reviews_table.js

```diff
@@ -0,0 +1,32 @@
+const TABLE_NAME = 'reviews';
+
+/**
+ * Create table reviews.
+ *
+ * @param   {object} knex
+ * @returns {Promise}
+ */
+export function up(knex) {
+  return knex.schema.createTable(TABLE_NAME, (table) => {
+    table.increments();
+    table.integer('customer_id').references('id').inTable('users');
```

Comment:
```suggestion
    table.integer('user_id').references('id').inTable('users');
```

Reply:
ah we can make user_id here for table name and id consistency

Reply:
I believe using `customer_id` is more appropriate here because users can only review a movie if they have rented it, making them customers in this context.

Reply:
that logic can change later but column name shouldn't change. In service level we can retrieve and rename it